islandora_batch.install

```diff
@@ -358,3 +358,12 @@ function islandora_batch_update_7104() {
     'not null' => FALSE,
   ));
 }
+
+/**
+ * Deprecates mods_to_dc.xsl
```

Comment:
Hi, this is new to me. Is this the standard way of notifying people about a file that is not in use anymore? I wonder if the wording could lead to people thinking they need to make a change or something, which is not true. This is more for developers depending on that file in other modules. Could you make something more re-assuring, like "If you are an user of this module no action is needed  (really), all good, be calm, don't snap!).. not the exact words. @willtp87 @adam-vessey you have some ideas for this?

Reply:
I have no idea what's standard, just copying language I've seen before. I can certainly update the text; no idea if "Deprecates" is the right term here.

Reply:
@bondjimbond me neither! Let's  wait for some wisdom coming from @willtp87 && @adam-vessey.

Reply:
Message updated; does that look all right?